src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -1979,6 +2076,7 @@ def _get_outbound_type(
                     else:
                         if (
                             self.get_load_balancer_managed_outbound_ip_count() or
+                            self.get_load_balancer_managed_outbound_ipv6_count() or
```

Comment:
This validation is not included in aks-preview

Reply:
I think that's an oversight then. we should raise an error if outbound type is UDR and there is IPv6 count defined similar to the _ip_count parameter. I added a unit test to cover this